indexing-service/src/main/java/io/druid/indexing/overlord/hrtr/HttpRemoteTaskRunner.java

```diff
@@ -376,36 +376,46 @@ private boolean runTaskOnWorker(
       // on a worker - this avoids overflowing a worker with tasks
       long waitMs = config.getTaskAssignmentTimeout().toStandardDuration().getMillis();
       long waitStart = System.currentTimeMillis();
+      boolean isTaskAssignmentTimedout = false;
```

Comment:
nit: isTaskAssignmentTimedOut (spelling: timed out is two words.)

Reply:
fixed :)